source/docs/software/mecanum-drive.rst

```diff
@@ -58,23 +58,26 @@ We want a positive X value to correlate to rightward strafing. If we refer back
    frontRightMotor.setPower(y - x - rx);
    backRightMotor.setPower(y + x - rx);
 
+.. note:: It's important to note that motors in FTC spin counterclockwise when given positive power by default (except for Neverest motors). In this case, you need to reverse the direction of the right drive motors so that they spin toward the same direction as the left drive motors when supplied with a positive power. This can be done simply by either reversing the direction with either :code:`DcMotor.setDirection(DcMotor.Direction.REVERSE)` or multiplying the right side powers by a negative number.
```

Comment:
```suggestion
.. important:: Motors in FTC spin counterclockwise when given positive power by default (except for NeveRest motors). In this case, you need to reverse the direction of the right drive motors so that they spin toward the same direction as the left drive motors when supplied with a positive power (for a drivetrain using NeveRests, reverse the right side instead). This can be done with :code:`DcMotor.setDirection(DcMotor.Direction.REVERSE)`.
```